evaluation/aider_bench/run_infer.py

```diff
@@ -59,6 +61,18 @@ def get_config(
         workspace_mount_path=None,
     )
     config.set_llm_config(metadata.llm_config)
+
+    # copy 'draft_editor' config if exists
+    config_copy = copy.deepcopy(config)
+    load_from_toml(config_copy)
+    if 'draft_editor' in config_copy.llms:
+        config.set_llm_config(config_copy.llms['draft_editor'], 'draft_editor')
+        if metadata.details is None:
+            metadata.details = {}
+        metadata.details['draft_editor'] = config_copy.llms[
+            'draft_editor'
+        ].to_safe_dict()
```

Comment:
This is a bit convoluted and potentially error-prone. I feel there should be a better way to do this (e.g. including `draft_editor` in `metadata`).

Reply:
yeah.. i've also been thinking about this in the back of my mind - we probably also need to figure out a way to set this in the UI as well, so it is probably a larger changes..

Reply:
I move this to a field in `LLMConfig` in newer commits - hopefully will make things clearer.